parse/token.go

```diff
@@ -33,6 +33,36 @@ const (
 	Illegal TokenType = iota
 
 	// Identifiers + literals
-	Ident // add, foobar, x, y, ...
-	Int   // 1343456
+	Ident  // add, foobar, x, y, ...
+	Int    // 1343456
+	String // "hello world"
+	Bool   // true, false
+
```

Comment:
Int    // 1343456
String // "hello world"
Bool   // true, false

Also we need int type tokenType